src/utils/setup/handle-create-page.js

```diff
@@ -24,7 +25,9 @@ const DEFAULT_FEATURED_LEARN_SLUGS = [
     'how-to/polymorphic-pattern',
 ];
 
-const STAGING_PAGES = ['/academia/educators/', '/media/'];
+const memoizedStagingPages = memoizerific(1)(
```

Comment:
I memoized this call since we would otherwise request Stitch once for each page that is built to get the Staging Pages. Now we only make one request.